libyang/schema.py

```diff
@@ -1107,3 +1124,4 @@ def _skip(node) -> bool:
 Node = SNode
 Rpc = SRpc
 RpcInOut = SRpcInOut
+Notif = SNotif
```

Comment:
These aliases were added for backward compatibility. No need to add one for `SNotif`.